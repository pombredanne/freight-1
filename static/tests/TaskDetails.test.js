import React from 'react';
import renderer from 'react-test-renderer';
import TaskDetails from '../components/TaskDetails.jsx';

test('TaskDetails Snapshot', () => {
  const params = {
    "app": "freight",
    "env": "production",
    "number": "798"
  }
  const component = renderer.create(
    (<TaskDetails params={params} />)
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
