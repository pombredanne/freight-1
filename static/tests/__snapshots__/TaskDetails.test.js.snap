// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskDetails Snapshot 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <div
    className="loading"
  >
    <div
      className="loading-mask"
    />
    <div
      className="loading-indicator"
    />
    <div
      className="loading-message"
    >
      Loading task details.
    </div>
  </div>
</div>
`;
